repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -0,0 +1,132 @@
+#########################
+# Gatling Configuration #
```

Comment:
I meant copying over a `gatling.conf` that looked like this:
```
#########################
# Gatling Configuration #
#########################

gatling {
  charting {
    indicators {
      percentile1 = 50      # Value for the 1st percentile to track in the reports, the console summary and Graphite
      percentile2 = 95      # Value for the 2nd percentile to track in the reports, the console summary and Graphite
      percentile3 = 99      # Value for the 3rd percentile to track in the reports, the console summary and Graphite
      percentile4 = 99.9    # Value for the 4th percentile to track in the reports, the console summary and Graphite
    }
  }
  data {
    writers = [console, file, graphite] # The list of DataWriters to which Gatling write simulation data (currently supported : console, file, graphite, jdbc)
    graphite {
      host = "influxdb.openrepose.org"  # The host where the Carbon server is located
      port = 12345                      # The port to which the Carbon server listens to (2003 is default for plaintext, 2004 is default for pickle)
    }
  }
}
```

Assuming that's allowed. I don't know if it's picky about needing the empty section names. That is:
```
gatling {
  core {
    extract {
      regex { }
    }
  }
...
```

Of note, the idiomatic way for the brutal approach is Ansible's `lineinfile` module, although its use is highly discouraged in favor of copying files and templates over instead.
http://docs.ansible.com/ansible/lineinfile_module.html